VisIt 3.5.0 no longer finds our eb HDF5.
Use SET_UP_THIRD_PARTY() to detect it properly again.
Author: Jarne Renders (Vrije Universiteit Brussel)

diff -ruN orig/src/CMake/FindHDF5.cmake new/src/CMake/FindHDF5.cmake
--- orig/src/CMake/FindHDF5.cmake	2026-07-10 16:39:39.738357389 +0200
+++ new/src/CMake/FindHDF5.cmake	2026-07-10 16:39:49.115232642 +0200
@@ -70,3 +70,6 @@
     file(APPEND ${fname} "\nset(HDF5_HL_LIB ${HDF5_HL_LIB})\n")
 endif()

+set(HDF5_DIR ${VISIT_HDF5_DIR})
+SET_UP_THIRD_PARTY(HDF5 LIBS hdf5 hdf5_hl)
+set(HDF5_FOUND TRUE)
